roles/openstack-setup/tasks/cinder.yml

```diff
@@ -19,6 +19,15 @@
     - (item.name in ['cinder','cinderv2','cinderv3'])
   run_once: true
 
+# FIXME: this task should be removed once old service is disabled for all envs
```

Comment:
We should be deleting old retired cinder scheduler service rather than leaving it disabled. We do something similar in https://github.com/blueboxgroup/ursula/blob/master/roles/nova-control/tasks/main.yml#L57